.gitignore

```diff
@@ -8,7 +8,7 @@
 .buildlog/
 .history
 .svn/
-*.env
+.env
```

Comment:
this change seems unnecessary. if you remove *, you will only remove files with name `.env`. Having `*.env` makes sure that any env file is ignored.
please revert is back